core/src/com/softwarearchitecture/game_server/buttons/GridLayout.java

```diff
@@ -0,0 +1,139 @@
+package com.softwarearchitecture.game_server.buttons;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.softwarearchitecture.GameApp;
+import com.softwarearchitecture.math.Rectangle;
+import com.softwarearchitecture.math.Vector2;
+
+public class GridLayout {
+
+    private int columns;
+    private int rows;
+
+    public GridLayout(int columns, int rows) {
+        this.columns = columns;
+        this.rows = rows;
+    }
+
+    public int getNumberOfColumns() {
+        return columns;
+    }
+
+    public int getNumberOfRows() {
+        return rows;
+    }
+
+    public Vector2 findCenter() {
+        return new Vector2(0.5f, 0.5f);
+    }
+
+    public Vector2 findCenterGrid() {
+        return new Vector2((float) columns / 2, (float) rows / 2);
+    }
+
+    public float getGridWidth() {
+        return 1f / columns * GameApp.WIDTH;
+    }
+
+    public float getGridHeight() {
+        return 1f / rows * GameApp.HEIGHT;
+    }
+
+    public Vector2 getGridPosition(int columnNumber, int rowNumber) {
+        return new Vector2((float) rowNumber / rows, (float) columnNumber / columns);
+    }
+
+    /**
+     * Finds the psositions the butons can be aligned in the center of the screen
+     * Parameters: numberOfButtons, rowNumber (where on the screen the buttons
+     * should be placed)
+     * Returns: List of Vector2 positions for the buttons
```

Comment:
To make the returns statement easier to see in the javadocs when hovering method signature use javadocs syntax like @return to achieve this